python/src/aac/plugins/rest_api/models/definition_model.py

```diff
@@ -1,27 +1,29 @@
 """Pydantic Version of the AaC Definition Class."""
-
+from typing import Optional
 from pydantic import BaseModel
 
+from aac.io.files.aac_file import AaCFile
 from aac.lang.definitions.definition import Definition
 
 
 class DefinitionModel(BaseModel):
     """REST API Model for the Definition class."""
     name: str
-    content: str
+    content: Optional[str] = ""
```

Comment:
If we're setting the default value to an empty string, do we really want this to be `Optional` (which would mean the default is `None`? I think we should pick one or the other, but using both seems incorrect.

Reply:
Good catch! Removed the default value.